tests/utils/test_common.py

```diff
@@ -18,6 +19,32 @@ def test_sort_dicts_by_keys():
     assert actual == expected
 
 
+@pytest.mark.parametrize(
+    "collection, possible_outputs",
+    [
+        (["a", "b", "c"], ["a, b and c"]),
+        (["a", "b"], ["a and b"]),
+        (["a"], ["a"]),
+        (
+            {"a", "b", "c"},
+            [
+                "a, b and c",
+                "a, c and b",
+                "b, a and c",
+                "b, c and a",
+                "c, a and b",
+                "c, b and a",
+            ],
+        ),
+        ({"a", "b"}, ["a and b", "b and a"]),
+        ({"a"}, ["a"]),
+    ],
+)
+def test_transform_collection_to_sentence(collection, possible_outputs):
+    actual = transform_collection_to_sentence(collection)
+    assert actual in possible_outputs
```

Comment:
?
```suggestion
    assert actual == possible_outputs
```

Reply:
The problem is that e.g. for a set we don't know the order. So I just listed all possible orders in `possible_outputs`, and the actual output needs to be one of them 😅

Reply:
haha :-D Ok then 👍